interface/themes/login_page.scss

```diff
@@ -28,14 +28,12 @@ body.login {
 }
 
 .login .container {
-  -webkit-border-radius: 10px 10px 10px 10px;
-  border-radius: 10px 10px 10px 10px;
+  border-radius: 10px;
   background: $white;
   margin-top: 75px;
   max-width: 785px;
   padding: 25px;
-  -webkit-box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
-  box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
+  box-shadow: 0 30px 60px 0 rgba(black,0.3);
```

Comment:
This should be $black

Reply:
I have tried it with and without the $. And neither of them work in the login.css. I tried all the major browsers and I do keep my browsers up to date.